app/helpers/layout_helper.rb

```diff
@@ -63,6 +63,20 @@ def title(page_title, page_header = nil)
     @page_header ||= page_header || page_title.to_s
   end
 
+  def edit_title(obj = nil, page_header: nil, attrbiture: nil)
```

Comment:
Why it combines keyword args with positional args if all are optional?

Reply:
I think the `obj` parameter shouldn't be optional, after all it's `Edit` page of a resource, do we have edit pages without any resource to edit?